src/libs/actions/Report.js

```diff
@@ -65,6 +65,7 @@ Onyx.connect({
 const currentReportData = {};
 Onyx.connect({
     key: ONYXKEYS.COLLECTION.REPORT,
+    waitForCollectionCallback: true,
```

Comment:
please explain why have you added waitForCollectionCallback flag in few places

Reply:
You can read more about it [here](https://github.com/Expensify/react-native-onyx). The gist is to update the callback with the collection once initially, not in pieces.

<img width="944" alt="Screenshot 2023-10-20 at 11 26 34 AM" src="https://github.com/Expensify/App/assets/47336142/c01a92cb-bf8a-456b-8c09-b3b614460bf7">